doc/api/core/helpers.rst

```diff
@@ -33,3 +33,7 @@ Convert to and from VTK Datatypes
 Fit Plane to Points
 ~~~~~~~~~~~~~~~~~~~
 .. automethod:: pyvista.helpers.fit_plane_to_points
+
+Textures
+~~~~~~~~
+.. automethod:: pyvista.core.objects.Texture
```

Comment:
For some reason this doesn't show up in the build artifact (haven't run the doc build locally yet).

This is the bottom of the "helpers" page:
![Screenshot from 2022-06-19 01-09-26](https://user-images.githubusercontent.com/17914410/174459742-9d687495-d8c3-471e-96db-5367413bc145.png)

Reply:
Updates locally on my end. I think it's just due to the doc cache.